dags/client_count_daily.py

```diff
@@ -0,0 +1,26 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2016, 8, 13),
+    'email': ['telemetry-alerts@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('client_count_daily', default_args=default_args, schedule_interval='@daily')
+
+client_count_view = EMRSparkOperator(
+    task_id="client_count_daily_view",
+    job_name="Client Count Daily View",
+    execution_timeout=timedelta(hours=10),
+    owner="frank@mozilla.com",
+    instance_count=20,
```

Comment:
instance_count can be reduced since we're only processing one day's data. Let's start with `5` and see how that performs.